radixdlt-core/radixdlt/src/main/java/org/radix/network/messaging/Message.java

```diff
@@ -96,7 +94,7 @@ protected Message() {
 		this(Time.currentTimestamp());
 	}
 
-	protected Message(long timestamp) {
+	private Message(long timestamp) {
 		this.timestamp = timestamp;

```

Comment:
Why it needs to be private?

Reply:
Good point. It is not used at all, so we can completely remove it.